pkg/ui/distccl/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+genrule(
+    name = "assets",
```

Comment:
Bad style for a rule to have the same name as an output file, so you should give the rule a different name (e.g. `genassets`).

Reply:
Done